### ISSUE TITLE
(Android) Crash (ConcurrentModificationException)

### ISSUE THREAD

**Issue by SudoPlz** [Instabug/instabug-reactnative#97]
Hey there, I received the following crash today in one of our production devices:

```
java.util.ConcurrentModificationException
    ArrayList.java:831 java.util.ArrayList$Itr.next
    UserEvent.java:65 com.instabug.library.user.a.a
    SessionManager.java:184 com.instabug.library.l.k
    SessionManager.java:138 com.instabug.library.l.c
    SessionManager.java:269 com.instabug.library.l.a
    CurrentActivityLifecycleChanged.java:25 com.instabug.library.tracking.b.onReceive
    LocalBroadcastManager.java:308 android.support.v4.content.LocalBroadcastManager.executePendingBroadcasts
    LocalBroadcastManager.java:46 android.support.v4.content.LocalBroadcastManager.access$000
    LocalBroadcastManager.java:118 android.support.v4.content.LocalBroadcastManager$1.handleMessage
    Handler.java:102 android.os.Handler.dispatchMessage
    Looper.java:154 android.os.Looper.loop
    ActivityThread.java:6317 android.app.ActivityThread.main
    Method.java:-2 java.lang.reflect.Method.invoke
    ZygoteInit.java:872 com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run
    ZygoteInit.java:762 com.android.internal.os.ZygoteInit.main
```

Is that stack-trace enough to get it fixed?
Is there anything else that I can do to help you without contacting our client?

Device: `LG MP450 Android 7.0 - API 24`
Version: `instabug-reactnative: "2.0.15"`

**Comment by SudoPlz** [comment]
Any news on that?

**Comment by ZiadAyman** [comment]
hey @SudoPlz 
Apologies for the late reply. 

Let me communicate this with our team and get back to you with updates.

**Comment by ZiadAyman** [comment]
Hey @SudoPlz 

Here is the fix for the crash.

npm istall https://github.com/Instabug/instabug-reactnative#hotfix/fix_android_crash 

Please let me know if there is anything else we need to fix from our side :)

**Comment by SudoPlz** [comment]
You're the best @ZiadAyman hmm, what official version would that crash be fixed on? 2.0.17?

**Comment by SudoPlz** [comment]
Hey there, any idea when that will be merged? I noticed 2.0.19 was released but that commit was not in there.

Thanks
